ui/components/email-badge.js

```diff
@@ -0,0 +1,18 @@
+import { XIcon } from '@heroicons/react/outline'
+
+export default function EmailBadge({ email, onRemove }) {
```

Comment:
Nit: should we generalize this to `Badge` and instead of `email` – you can render the `children` prop below